src/Faker/Provider/az_AZ/Address.php

```diff
@@ -0,0 +1,137 @@
+        'Litva', 'Livan', 'Liviya', 'Lixtenşteyn', 'Lüksemburq', 'Macarıstan', 'Madaqaskar', 'Makao',
+        'Makedoniya', 'Malavi', 'Malayziya', 'Maldiv adaları', 'Mali', 'Malta', 'Marşall adaları',
+        'Mavriki', 'Mavritaniya', 'Mayotta', 'Meksika', 'Men adası', 'Mərakeş', 'MAR', 'Mikroneziya',
+        'Milad adası', 'Misir', 'Myanma', 'Moldova', 'Monako', 'Monqolustan', 'Montserrat', 'Mozambik',
+        'Müqəddəs Yelena, Askenson və Tristan-da-Kunya adaları', 'Namibiya', 'Nauru', 'Nepal', 'Niderland',
+        'Niderland Antil adaları', 'Niger', 'Nigeriya', 'Nikaraqua', 'Niue', 'Norfolk adası', 'Norveç',
+        'Oman', 'Özbəkistan', 'Pakistan', 'Palau', 'Panama', 'Papua-Yeni Qvineya', 'Paraqvay', 'Peru',
+        'Pitkern adaları', 'Polşa', 'Portuqaliya', 'Prednestroviya', 'Puerto-Riko', 'Ruanda', 'Rumıniya',
+        'Rusiya', 'Salvador', 'Samoa', 'San-Marino', 'San-Tome və Prinsipi', 'Seneqal', 'Sen-Bartelemi',
+        'Sent-Kits və Nevis', 'Sent-Lüsiya', 'Sen-Marten', 'Sen-Pyer və Mikelon', 'Sent-Vinsent və Qrenadina',
+        'Serbiya', 'Seyşel adaları', 'Səudiyyə Ərəbistanı', 'Sinqapur', 'Slovakiya', 'Sloveniya', 'Solomon adaları',
+        'Somali', 'Somalilend', 'Sudan', 'Surinam', 'Suriya', 'Svazilend', 'Syerra-Leone', 'Şərqi Timor',
+        'Şimali Marian adaları', 'Şpisbergen və Yan-Mayen', 'Şri-Lanka', 'Tacikistan', 'Tanzaniya', 'Tailand',
+        'Çin Respublikası', 'Törks və Kaykos adaları', 'Tokelau', 'Tonqa', 'Toqo', 'Trinidad və Tobaqo',
+        'Tunis', 'Tuvalu', 'Türkiyə', 'Türkmənistan', 'Ukrayna', 'Uollis və Futuna', 'Uqanda', 'Uruqvay',
+        'Vanuatu', 'Vatikan', 'Venesuela', 'Amerika Virgin adaları', 'Britaniya Virgin adaları',
+        'Vyetnam', 'Yamayka', 'Yaponiya', 'Yeni Kaledoniya', 'Yeni Zelandiya', 'Yəmən', 'Yunanıstan', 'Zambiya', 'Zimbabve'
+    );
+
+    protected static $region = array(
+    );
+
+    protected static $city = array(
+        'Ağcabədi', 'Ağdam', 'Ağdaş', 'Ağdərə', 'Ağstafa', 'Ağsu', 'Astara',
+        'Bakı', 'Balakən', 'Beyləqan', 'Bərdə', 'Biləsuvar', 'Cəbrayıl',
+        'Cəlilabad', 'Culfa', 'Daşkəsən', 'Dəliməmmədli', 'Füzuli', 'Gədəbəy',
+        'Gəncə', 'Goranboy', 'Göyçay', 'Göygöl', 'Göytəpə', 'Hacıqabul',
+        'Horadiz', 'Xaçmaz', 'Xankəndi', 'Xocalı', 'Xocavənd', 'Xırdalan',
+        'Xızı', 'Xudat', 'İmişli', 'İsmayıllı', 'Kəlbəcər', 'Kürdəmir', 'Qax',
+        'Qazax', 'Qəbələ', 'Qobustan', 'Qovlar', 'Quba', 'Qubadlı', 'Qusar',
+        'Laçın', 'Lerik', 'Lənkəran', 'Liman', 'Masallı', 'Mingəçevir',
+        'Naftalan', 'Naxçıvan (şəhər)', 'Neftçala', 'Oğuz', 'Ordubad',
+        'Saatlı', 'Sabirabad', 'Salyan', 'Samux', 'Siyəzən', 'Sumqayıt',
+        'Şabran', 'Şahbuz', 'Şamaxı', 'Şəki', 'Şəmkir', 'Şərur', 'Şirvan',
+        'Şuşa', 'Tərtər', 'Tovuz', 'Ucar', 'Yardımlı', 'Yevlax', 'Zaqatala',
+        'Zəngilan', 'Zərdab',
+    );
+
+    protected static $street = array(
```

Comment:
this dataset is too small, the generated data will repeat too much. Please provide a larger dataset.